src/plugins/orangehrmLDAPAuthenticationPlugin/config/routes.yaml

```
@@ -12,3 +12,11 @@ apiv2_admin_ldap_test_connection:
   methods: [ POST ]
   defaults:
     _api: OrangeHRM\LDAP\Api\LDAPTestConnectionAPI
+
+apiv2_admin_ldap_user_sync:
+  path: /api/v2/admin/ldap/user-sync
```

```suggestion
  path: /api/v2/admin/ldap/sync
```
or `/api/v2/admin/ldap/sync-users`
:thinking:

---

shall we keep this for the moment. Once Chamara finished with the FE, he can remove the mock API and change the path